syncer/db.go

```diff
@@ -159,7 +159,7 @@ func (conn *Conn) executeSQLWithIgnore(tctx *tcontext.Context, ignoreError func(
 		FirstRetryDuration: retryTimeout,
 		BackoffStrategy:    retry.Stable,
 		IsRetryableFn: func(retryTime int, err error) bool {
-			if retry.IsRetryableError(err) {
+			if retry.IsRetryableErrorFastFailFilter(err) {
```

Comment:
only needing to replace `IsRetryableError` with `IsRetryableErrorFastFailFilter` in sync's `execute` SQL, right?

Reply:
yes. have effect on DDL execution in sync unit only